docker-compose.yaml

```diff
@@ -34,7 +34,31 @@ services:
       - /tmp/.X11-unix:/tmp/.X11-unix
       - ~/.Xauthority:/root/.Xauthority
     working_dir: "/project/TrustedComputeFramework"
-    entrypoint: "bash -c \"\
-        source scripts/tcs_startup.sh -y \
-        && tail -f /dev/null \""
+    command: |
+      bash -c "
+        source scripts/tcs_startup.sh -y
+        tail -f /dev/null
+      "
     stop_signal: SIGKILL
+    depends_on:
+      - lmdb
+
+  lmdb:
```

Comment:
Use a less general name than `lmdb` to avoid conflicts. Something more specific like `avalon-lmdb` or `tcf-lmdb`

Reply:
Changed.